cmd/kubectl-gadget/utils/trace.go

```diff
@@ -85,12 +85,12 @@ type TraceConfig struct {
 	// TraceInitialState is the state in which the trace should be after its
 	// creation.
 	// This field is only used by "multi-rounds gadgets" like biolatency.
-	TraceInitialState string
+	TraceInitialState gadgetv1alpha1.TraceState
 
 	// CommonFlags is used to hold parameters given on the command line interface.
 	CommonFlags *CommonFlags
 
-	// Parameters is used to pass specific gadget configurations.
+	// Parameters are used to pass specific gadget configurations.
```

Comment:
nit: I think the current use of the singular is fine as it refers to `Parameters` as a whole.